src/containers/SearchResults.js

```diff
@@ -17,6 +18,8 @@ import {
  * @private
  */
 const mapStateToProps = (state, { companionWindowId, windowId }) => ({
+  searchAnnotations:
+    getSearchAnnotationForCompanionWindow(state, { companionWindowId, windowId }).resources,
```

Comment:
Seems like that `getSearchAnnotationForCompanionWindow` can return `undefined` which will cause an issue with opening the panel.